data/channels.csv

```diff
@@ -16997,185 +16997,185 @@ KUWBLD3.us,KUWB-LD3,,Defy TV,Nexstar Media,US,US-UT,Bloomington,s/US-UT,eng,ente
-KVMDDT7.us,KVMD-DT7,,Infomercials,,US,US-CA,Twentynine Palms,s/US-CA,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/f/f6/As_seen_on_TV.svg/512px-As_seen_on_TV.svg.png
-KVMDDT8.us,KVMD-DT8,,CCTV-4,KVMD TV,US,US-CA,Twentynine Palms,c/US,zho,general,FALSE,,,,https://kvmdtv.com/,https://static.wikia.nocookie.net/logopedia/images/3/38/Cctv_4_america.png/revision/latest/scale-to-width-down/250?cb=20200119091533
-KVMDDT9.us,KVMD-DT9,,CGTN,,US,US-CA,Twentynine Palms,s/US-CA,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/8/81/CGTN.svg/512px-CGTN.svg.png
-KVMEDT1.us,KVME-DT1,,Jewelry TV,,US,US-CA,Bishop,s/US-CA,eng,,FALSE,,,,https://www.jtv.com/,https://upload.wikimedia.org/wikipedia/en/5/50/Jewelry_television_logo.png
-KVMEDT2.us,KVME-DT2,,H&I,,US,US-CA,Bishop,s/US-CA,eng,,FALSE,,,,https://www.handitv.com/,https://upload.wikimedia.org/wikipedia/en/thumb/9/9e/Heroes_%26_Icons_logo.svg/512px-Heroes_%26_Icons_logo.svg.png
-KVMMCD2.us,KVMM-CD2,,Infomercials,,US,US-CA,Santa Barbara,s/US-CA,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/f/f6/As_seen_on_TV.svg/512px-As_seen_on_TV.svg.png
+KVIADT1.us,KVIA-DT1,,ABC,News-Press & Gazette Company,US,US-TX,El Paso,s/US-TX,eng,general,FALSE,1956-09-01,,,http://www.kvia.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/2/2f/ABC-2021-LOGO.svg/512px-ABC-2021-LOGO.svg.png
+KVIADT2.us,KVIA-DT2,,CW,News-Press & Gazette Company,US,US-TX,El Paso,s/US-TX,eng,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/The_CW.svg/512px-The_CW.svg.png
+KVIADT3.us,KVIA-DT3,,ION,News-Press & Gazette Company,US,US-TX,El Paso,s/US-TX,eng,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/2/28/Ion_logo.svg/512px-Ion_logo.svg.png
+KVIADT4.us,KVIA-DT4,,QVC,News-Press & Gazette Company,US,US-TX,El Paso,s/US-TX,eng,shop,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/a/aa/Azteca_America_2011_logo.png
+KVIEDT1.us,KVIE-DT1,,PBS,KVIE Inc,US,US-CA,Sacramento,s/US-CA,eng,education,FALSE,1959-02-23,,,http://www.kvie.org/,https://upload.wikimedia.org/wikipedia/en/thumb/3/33/PBS_logo.svg/512px-PBS_logo.svg.png
+KVIEDT2.us,KVIE-DT2,,KVIE2,KVIE Inc,US,US-CA,Sacramento,s/US-CA,eng,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/6/6b/PBSworld.png
+KVIEDT3.us,KVIE-DT3,,World,KVIE Inc,US,US-CA,Sacramento,s/US-CA,eng,documentary,FALSE,,,,,https://upload.wikimedia.org/wikipedia/en/thumb/7/76/PBS_Kids_Logo.svg/512px-PBS_Kids_Logo.svg.png
+KVIHDT1.us,KVIH-DT1,,ABC,Sinclair,US,US-NM,Clovis,s/US-NM,eng,general,FALSE,1978-09-25,,,http://www.kvii.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/2/2f/ABC-2021-LOGO.svg/512px-ABC-2021-LOGO.svg.png
+KVIHDT2.us,KVIH-DT2,,CW+,Sinclair,US,US-NM,Clovis,s/US-NM,eng,entertainment,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/The_CW.svg/512px-The_CW.svg.png

```

Comment:
Make sure the CW logo isn't used for the CW+ channel, as it might create confusion among users.